execCommand.html

```diff
@@ -639,68 +640,44 @@ <h2 id="methods-to-query-and-execute-commands">Methods to query and
         bugs. The spec is not finalized yet and can still be easily
         changed.</strong></p>
         <pre class="idl">
-[Constructor(DOMString <var title="">type</var>, optional <a href=
-"#editingbeforeinputeventinit">EditingBeforeInputEventInit</a> <var title=
-"">eventInitDict</var>)]
-interface <dfn id=
-"editingbeforeinputevent">EditingBeforeInputEvent</dfn> : <a class="external"
-data-anolis-spec="dom" href="http://dom.spec.whatwg.org/#event">Event</a> {
-  readonly attribute DOMString <a href="#dom-editingbeforeinputevent-command"
-title="dom-EditingBeforeInputEvent-command">command</a>;
-  readonly attribute DOMString <a href="#dom-editingbeforeinputevent-value"
-title="dom-EditingBeforeInputEvent-value">value</a>;
-};
-
-dictionary <dfn id=
-"editingbeforeinputeventinit">EditingBeforeInputEventInit</dfn> : <a class=
-"external" data-anolis-spec="dom" href=
-"http://dom.spec.whatwg.org/#eventinit">EventInit</a> {
-  DOMString command;
-  DOMString value;
-};
-
-[Constructor(DOMString <var title="">type</var>, optional <a href=
-"#editinginputeventinit">EditingInputEventInit</a> <var title=
-"">eventInitDict</var>)]
-interface <dfn id="editinginputevent">EditingInputEvent</dfn> : <a class=
-"external" data-anolis-spec="dom" href=
-"http://dom.spec.whatwg.org/#event">Event</a> {
-  readonly attribute DOMString <a href="#dom-editinginputevent-command" title=
-"dom-EditingInputEvent-command">command</a>;
-  readonly attribute DOMString <a href="#dom-editinginputevent-value" title=
-"dom-EditingInputEvent-value">value</a>;
-};
-
-dictionary <dfn id=
-"editinginputeventinit">EditingInputEventInit</dfn> : <a class="external"
-data-anolis-spec="dom" href=
-"http://dom.spec.whatwg.org/#eventinit">EventInit</a> {
-  DOMString command;
-  DOMString value;
-};
-</pre>
+            [Exposed=Window,
+             Constructor(DOMString type, optional EditingBeforeInputEventInit eventInitDict = {})]
+            interface EditingBeforeInputEvent : Event {
+              readonly attribute DOMString command;
+              readonly attribute DOMString value;
+            };
+            
+            dictionary EditingBeforeInputEventInit : EventInit {
+              DOMString command;
+              DOMString value;
+            };
+
+            [Exposed=Window,
+             Constructor(DOMString type, optional EditingInputEventInit eventInitDict = {})]
+            interface EditingInputEvent : Event {
+              readonly attribute DOMString command;
+              readonly attribute DOMString value;
+            };
+            
+            dictionary EditingInputEventInit : EventInit {
+              DOMString command;
```

Comment:
```suggestion
              DOMString command = "";
```

Reply:
Committing the suggestion somehow didn't work, so I did them manually.